14-microbiome-diversity.Rmd

```diff
@@ -150,15 +194,27 @@ se <- runTSNE(se, name = "TSNE", exprs_values = "counts", ncomponents = 3)
 plotReducedDim(se, "TSNE", colour_by = "SampleType", ncomponents = c(1:3))
 ```
 
-## Community comparisons
 
-TODO: Introduction
+
+## Community comparisons [TODO combine with the material above for simplicity?]
```

Comment:
I personally like the introduction with some more applied example at this point. Otherwise feel free to restructure the chapter